pkg/media/gstreamer.go

```diff
@@ -258,18 +262,24 @@ func (mm *MediaManager) TestSource(ctx context.Context, ms MediaSigner) error {
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 	go func() {
+		if err := pipeline.BlockSetState(gst.StateNull); err != nil {
+			log.Log(ctx, "failed to set pipline state", "error", err)
+		}
 		<-ctx.Done()
```

Comment:
Oop, definitely don't want this, we're gonna shut down the pipeline before anything happens